c_api.h

```diff
@@ -18,9 +99,15 @@ SST_ctx_t *init_SST(char *config_path);
 session_key_list_t *get_session_key(SST_ctx_t *ctx,
                                     session_key_list_t *existing_s_key_list);
 
-// Connect with other entity such as entity servers using secure session key.
+// Connect to entity_server using the session key. This function can be called
+// after the connect() function, and uses the user's socket.
+SST_session_ctx_t *secure_connect_to_server_with_socket(session_key_t *s_key,
+                                                        int sock);
+
+// Connect with other entity such as entity servers using the session key. This
+// function contains the connect() function, and uses the
+// secure_connect_to_server_with_socket() function.
 // @param s_key session key struct received by Auth
-// @param ctx config struct obtained from load_config()
```

Comment:
Please add back the explanation of ctx param.

Reply:
Fixed b1433f6d01bab31b6e02c925559a72036debdbb4.